Tests/tests/test_OCIContainer.cpp

```diff
@@ -8,13 +8,17 @@
 using namespace WPEFramework;
 using ::testing::NiceMock;
 
-namespace{
-    const int32_t descriptor1 = 91;
-    const int32_t descriptor2 = 94;
-    const string containerId1 = "com.bskyb.epgui";
-    const string containerId2 = "Netflix";
-    const int stateRunning = 2;
-    const int listenerId = 5;
+namespace {
+
+    enum ContainerState {
```

Comment:
Looks like you have the same enum as IDobbyProxyEvents::ContainerState. Why not just use it?

Reply:
I have applied this change if you could review whenever you get a chance please